downstream/modules/vault-aap/proc-vault-configuring-kv2-secret.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="vault-configuring-kv2-secret"]
+
+= Configuring the `hashicorp.vault.kv2_secret` module
+
+[role="_abstract"]
+
+The `hashicorp.vault.kv2_secret` module performs Create, Update, and Delete operations on KV2 secrets through a unified interface.
+
+The corresponding `community.hashi_vault` modules are:
+
+* **`community.hashi_vault.vault_kv2_write`** - Write KV2 secrets.
+* **`community.hashi_vault.vault_kv2_delete`** - Delete KV2 secrets.
+
+.Prerequisites
+
+* Install the {PlatformNameShort} certified `hashicorp.vault` collection.
+
+.Procedure
+
+. Replicate your automation tasks from both of the `community.hashi_vault` modules to the following `hashicorp.vault.kv2_secret` parameters.  The `hashicorp.vault.kv2_secret` parameters are similar to `community.hashi_vault`. For examples, see _Migration examples for the `hashicorp.vault.kv2_secret` module_.
++
+----
+auth_method:
+ description: Authentication method to use
+ type: str
+ choices: [token]
+ default: token
+ required: false
+
+cas:
+ description: Perform a check-and-set operation.
+ type: int
+ required: false
+
+data:
+ description: KV2 secret data to write.
+ type: dict
+ required: true
+
+engine_mount_point:
+ description: The path where the secret backend is mounted.
+ type: str
+ default: secret
+ required: false
+ aliases: [secret_mount_path]
+
+namespace:
+ description: Vault namespace where secrets reside.
+ type: str
+ default: admin
+ aliases: [vault_namespace]
+
+path:
+ description: Vault KVv2 path to be written to.
+ type: str
+ required: true
+ aliases: [secret_path]
+
+url:
+ description: URL of the Vault service
+ type: str
+ aliases: [vault_address]
+ required: true
+
+versions:
+ description: One or more versions of the secret to delete.
+ type: list of int
+ required: false
+
+state:
+ description: Desired state of the secret
+ type: str
+ choices: [present, absent]
+ default: present
+----
+
+. You must add the `state` parameter to the `hashicorp.vault.kv2_secret` module, as shown above. Valid options are:
```

Comment:
This list originally had "secret" in the table under "Notes" But there is no option to set "secret" in the code example. If we need to add "secret" to this list of options, please let me know what the description should be for it. Thanks.